src/Bicep.Cli/Arguments/DecompileArguments.cs

```diff
@@ -84,6 +86,15 @@ public DecompileArguments(string[] args) : base(Constants.Command.Decompile)
                     throw new CommandLineException(string.Format(CliResources.DirectoryDoesNotExistFormat, outputDir));
                 }
             }
+
+            var inputPath = PathHelper.ResolvePath(InputFile);
```

Comment:
@majastrz I don't like this code duplication.  I think the code in DecompileCommand::RunAsync:
```cs
        public async Task<int> RunAsync(DecompileArguments args)
        {
            logger.LogWarning(CliResources.DecompilerDisclaimerMessage);

            var inputPath = PathHelper.ResolvePath(args.InputFile);

            static string DefaultOutputPath(string path) => PathHelper.GetDefaultDecompileOutputPath(path);

            var outputPath = PathHelper.ResolveDefaultOutputPath(inputPath, args.OutputDir, args.OutputFile, DefaultOutputPath);
```
belongs here in DecompileArguments (where the resolved paths are the ones exposed).  Same for BuildCommand.

Reply:
@vlahane Are you willing to do the suggested refactoring?  If not, I'll consider it post-merge.  Thakns.

Reply:
done some change please review. Thanks 😃